forms-flow-theme/scss/_button.scss

```diff
@@ -137,44 +131,183 @@
+      background-color: var(--base-base-20, #CFD4FC) !important;
+      font-weight: var(--font-weight-xl) !important;
+    }
+    &:focus {
+      outline: none !important;
+    }
   }
+  &.btn-dark{
+    background-color: var(--ff-gray-700, #494E50) !important;
+
+    &:hover {
+      background-color: var(--ff-gray-600, #61686B) !important;
+      box-shadow: var(--shadow-md) !important;
+    }
+
+    &:active {
+      background-color: var(--ff-white, #ffffff) !important;
+      color: var(--ff-gray-800, #303436) !important;
+      border: 1px solid var(--ff-gray-700, #494E50) !important;
+    }
+
+        // Dropdown item styles for btn-dark
+        & + .dropdown-toggle + .dropdown-menu {
+          border: 1px solid var(--ff-gray-700, #494E50) !important;
+          .dropdown-item {
+            color: var(--ff-gray-800, #303436) !important;
+            &:hover {
+              background-color: transparent !important; 
+              color: var(--ff-gray-800, #303436) !important;
+            }
+          }
+        }
+  }
+  &.btn-sm {
+    font-size: var(--font-size-xs) !important;
+    padding: 0.5rem var(--spacer-100, 1rem) !important;
+    font-weight: var(--font-weight-xl) !important;
+    border-radius: var(--radius-0100) !important;
+
+    // Dropdown item styles for btn-sm
+    ~ .dropdown-menu .dropdown-item {
+      font-size: var(--font-size-xs);
+      font-weight: var(--font-weight-xl) !important;
+      line-height: var(--text-line-height); 
+      padding: 0.5rem var(--spacer-100, 1rem) !important;
+    }
+  }
+  &.btn-md {
+    font-size: var(--font-size-sm) !important;
+    padding: 0.75rem var(--spacer-150, 1.5rem) !important;
+    font-weight: var(--font-weight-xl) !important;
+    border-radius: var(--radius-0125) !important;
+
+    // Dropdown item styles for btn-md
+    ~ .dropdown-menu .dropdown-item {
+      font-size: var(--font-size-sm);
+      font-weight: var(--font-weight-xl) !important;
+      line-height: var(--text-line-height); 
+      padding: 0.75rem var(--spacer-150, 1.5rem) !important;
+    }
+  }
+
+  &.btn-lg {
+    font-size: var(--font-size-sm) !important;
+    padding: 1rem var(--spacer-200, 2rem) !important;
+    font-weight: var(--font-weight-xl) !important;
+    border-radius: var(--radius-0150) !important;
+
+    // Dropdown item styles for btn-lg
+    ~ .dropdown-menu .dropdown-item {
+      font-size: var(--font-size-sm);
+      font-weight: var(--font-weight-xl) !important;
+      line-height: var(--text-line-height); 
+      padding: 1rem var(--spacer-200, 2rem) !important;
+    }
+  }
+
 }
 
-.button,
-.btn.btn-secondary {
-  line-height: 1.15 !important;
+.dropdown.btn-group {
+  .btn {
+    border-radius: 0 !important;
 }
-.btn.btn-secondary {
-  border: none;
-  color:#253DF4 !important;
-  font-size: 14px !important;
-  font-weight: bold !important;
-  border-radius: 24px !important;
-  background: var(--base-base-10, #E7E9FE) !important;
+}
+.btn-group > .btn:first-child,
```

Comment:
Please do Nesting while writing scss selectors wherever not implemented for easy maintenance